source/_integrations/matter.markdown

```diff
@@ -25,180 +25,237 @@ ha_integration_type: integration
 
-1)  Open the device in Google Home and press the settings button (jog wheel) in the top right.
-2)  Click `Linked Matter apps and services`.
-3)  Press the button `Link apps and services` to link the device to Home Assistant.
-4)  Choose Home Assistant from the list, you are redirected to the Home Assistant Companion app now. Press `Add device`.
-5)  Your device will now be added to Home Assistant. When the process finishes, you're redirected to the device page in Home Assistant.
+1.  Open the device in Google Home and press the settings button (jog wheel) in the top right.
+2.  Click `Linked Matter apps and services`.
+3.  Press the button `Link apps and services` to link the device to Home Assistant.
+4.  Choose Home Assistant from the list, you are redirected to the Home Assistant Companion app now. Press `Add device`.
+5.  Your device will now be added to Home Assistant. When the process finishes, you're redirected to the device page in Home Assistant.
 
 <lite-youtube videoid="-B4WWevd2JI" videotitle="Share Matter device from Google Home to Home Assistant"></lite-youtube>
 
-## Device specific instructions
+<p class='note'>
+At this time it is not yet possible to share a device from Home Assistant to another platform. This feature should be added after the Matter SDK 1.1 is released.
+</p>
 
-Because availability of actual Matter devices is sparse and proper HOWTO documentation even more, we provide a few step by step instructions for devices we have currently tested.
+## Experiment with Matter using a ESP32 dev board
 
-### TP-Link Tapo P125M (power plug)
+You do not yet have any Matter compatible hardware but you do like to try it out or maybe create your own DIY Matter device ? We have [prepared a page for you](https://nabucasa.github.io/matter-example-apps/) where you can easily flash Matter firmware to a supported ESP32 development board. We actually recommend the M5 Stamp C3 device running the Lightning app.
 
-This device runs Matter out of the box, so you can add it directly to the controller(s) of your choice!
+NOTE for Android users: You need to follow the instructions at the bottom of the page to add the test device to the Google developer console, otherwise commissioning will fail. iOS users will not have this issue but they will get a prompt during commissioning asking if you trust the development device.
 
-Look for the M addition in the model name, a device without the M (regular P125) is not Matter compliant. This device is available in the US only.
+1. Make sure you are using the Google Chrome or Microsoft Edge browser.
+2. Open https://nabucasa.github.io/matter-example-apps/
+3. Attach the ESP32 device using an USB cable.
+4. Click the radiobutton next to the example you like to setup, in case of an M5 Stamp, click `Lightning app for M5STAMP C3`.
+5. Press `Connect`.
+6. In the popup dialog that appears choose the correct serial device, in most cases this will be something like "cu-usbserial" or alike.
```

Comment:
```suggestion
6. In the popup dialog that appears, choose the correct serial device, in most cases this will be something like "cu-usbserial" or alike.
```